[BUG] - unresolved import `plotters_bitmap`

**Describe the bug**
Unable to build plotters in another project that is using it.

**To Reproduce**
`git clone https://github.com/diem/diem.git`
`cargo install --path shuffle/cli`

**Error**
```
   Compiling plotters v0.3.3
error[E0432]: unresolved import `plotters_bitmap`
 --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/plotters-0.3.3/src/evcxr.rs:4:5
  |
4 | use plotters_bitmap::BitMapBackend;
  |     ^^^^^^^^^^^^^^^ use of undeclared crate or module `plotters_bitmap`
```

**Version Information**
Ubuntu 20.04

--- a/plotters/src/evcxr.rs
+++ b/plotters/src/evcxr.rs
@@ -1,8 +1,10 @@
 use crate::coord::Shift;
 use crate::drawing::{DrawingArea, IntoDrawingArea};
 use plotters_backend::DrawingBackend;
-use plotters_bitmap::BitMapBackend;
 use plotters_svg::SVGBackend;
+
+#[cfg(feature = "evcxr_bitmap")]
+use plotters_bitmap::BitMapBackend;
 
 /// The wrapper for the generated SVG
 pub struct SVGWrapper(String, String);
@@ -44,7 +46,7 @@ pub fn evcxr_figure<
 }
 
 /// Start drawing an evcxr figure
-#[cfg(all(feature = "evcxr_bitmap"))]
+#[cfg(feature = "evcxr_bitmap")]
 pub fn evcxr_bitmap_figure<
     Draw: FnOnce(DrawingArea<BitMapBackend, Shift>) -> Result<(), Box<dyn std::error::Error>>,
 >(

